crates/primitives/src/constants/mod.rs

```diff
@@ -67,6 +67,11 @@ pub const EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u64 = 2;
 #[cfg(feature = "optimism")]
 pub const OP_MAINNET_EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 50;
 
+/// Base fee max change denominator for Optimism Mainnet as defined in the Optimism Canyon
+/// hardfork.
+#[cfg(feature = "optimism")]
+pub const OP_MAINNET_EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR_CANYON: u64 = 250;
```

Comment:
nit: no longer the default
```suggestion
pub const OP_MAINNET_EIP1559_CANYON_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 250;
```